modules/administration/pages/image-management.adoc

```diff
@@ -187,46 +162,39 @@ If your container image tag is in a format such as `myproject/myimage`, make sur
 ----
 https://github.com/USER/project.git#branchname:folder
 ----
-
++
 * GitHub organization project repository
 +
 ----
 https://github.com/ORG/project.git#branchname:folder
 ----
-
++
```

Comment:
this one?

Reply:
This section was written before, some changes may have only been added to align things. I think it's worth a deeper look and reformatting, but only if time allows. For now, it works as is.